versioned_docs/version-7.x/stack-navigator.md

```diff
@@ -244,26 +259,47 @@ On Web, the height of the screen isn't limited to the height of the viewport. Th
 This is shortcut option which configures several options to configure the style for rendering and transitions:
 
 - `card`: Use the default OS animations for iOS and Android screen transitions.
+  <video playsInline autoPlay muted loop>
+    <source src="/assets/7.x/stack/presentation-card.mp4" />
+  </video>
+
 - `modal`: Use Modal animations. This changes a few things:
+
   - Sets `headerMode` to `screen` for the screen unless specified otherwise.
   - Changes the screen animation to match the platform behavior for modals.
+  <video playsInline autoPlay muted loop>
+    <source src="/assets/7.x/stack/presentation-modal.mp4" />
+  </video>
+
 - `transparentModal`: Similar to `modal`. This changes following things:
+
   - Sets `headerMode` to `screen` for the screen unless specified otherwise.
   - Sets background color of the screen to transparent, so previous screen is visible
   - Adjusts the `detachPreviousScreen` option so that the previous screen stays rendered.
   - Prevents the previous screen from animating from its last position.
   - Changes the screen animation to a vertical slide animation.
 
-See [Transparent modals](#transparent-modals) for more details on how to customize `transparentModal`.
+  See [Transparent modals](#transparent-modals) for more details on how to customize `transparentModal`.
+
+  <video playsInline autoPlay muted loop>
+    <source src="/assets/7.x/stack/presentation-transparentModal.mp4" />
+  </video>
```

Comment:
I don't know if this particular video example looks good 😕 Maybe we could set transparent background to `white`, just so it won't show what's behind the transparent modal? Or is it intentional?

Reply:
It is intentional, similar to: [Native Stack](https://reactnavigation.org/docs/7.x/native-stack-navigator#options)'s transparentModal presentation

Reply:
Agree with @tboba. This is not how `transparentModal` is intended to be used. Check this screen in our example app which is closer to the intended use case https://react-navigation-example.netlify.app/stack-transparent/article/gandalf